app/js/controllers/service.js

```diff
@@ -3,12 +3,27 @@
 import { Controller } from 'components/fxos-mvc/dist/mvc';
 
 import Service from 'js/views/service';
+import CameraService from 'js/views/camera-service';
 
 export default class ServiceController extends Controller {
   main(id) {
```

Comment:
Do you have any example when view itself can render another view to demonstrate what you mean? Sorry I'm quite new React JS?